specification/09-coveio_abi.adoc

```diff
@@ -40,8 +46,62 @@ upstream Root Port. The configured IDE link must use the selective IDE stream
 identified by `stream_id`.
 
 [#sbi_covh_disconnect_device()]
-=== Device Disconnection (FID TBD)
+===== Device Disconnection (FID TBD)
 [source, C]
 -----
 struct sbiret sbi_covh_disconnect_device()(unsigned long device_id);
 -----
+
+==== Device Interface Assignment
+
+
+=== CoVE IO Guest Extension
+
+==== Physical Device Query
+
+[#sbi_covg_get_device_link()]
+===== Device Link Status (FID TBD)
+[source, C]
+-----
+struct sbiret sbi_covg_get_device_link(unsigned long device_id);
+-----
+
+[#sbi_covg_get_device_certificate()]
+===== Device Identity Certificate (FID TBD)
+[source, C]
+-----
+struct sbiret sbi_covg_get_device_certificate(unsigned long device_id);
+-----
+
+[#sbi_covg_get_device_measurements()]
+===== Device Measurements (FID TBD)
+[source, C]
+-----
+struct sbiret sbi_covg_get_device_measurements(unsigned long device_id);
+-----
+
+[#sbi_covg_get_device_update_attrs()]
+===== Device Firmware Update Atttibutes (FID TBD)
```

Comment:
suggest to remove this API.

It can be combined into `sbi_covg_get_interface_report`

Reply:
maybe sbi_covg_get_device_spdm_attrs() and sbi_covg_get_device_tdisp_attrs().

Reply:
I kept the `sbi_covg_get_interface_report` as I think it provides the TDISP attributes we need.
I replaced `sbi_covg_get_device_update_attrs()`with `sbi_covg_get_device_spdm_attrs()`.